typescript/sdk/src/deploy/middleware/LiquidityLayerApp.ts

```diff
@@ -75,6 +73,44 @@ export class LiquidityLayerApp<
     return response.result.map((_: any) => _.transactionHash).flat();
   }
 
+  async fetchPortalBridgeTransactions(chain: Chain): Promise<string[]> {
```

Comment:
Just to make sure I'm reading this correctly:

This will return a finite amount of tx hashes (say the last 100 or something), so each iteration of the portal-relayer we'll likely be re-evaluating a bunch of Portal messages that have already been processed?

(not saying this is necessarily bad)

Reply:
Yeah did the same with circle. Really should keep state on the relayer but wanted to punt this for now